Envelope_3/include/CGAL/Env_sphere_traits_3.h

```diff
@@ -1080,7 +1080,7 @@ class Env_sphere_traits_3 : public ConicTraits_2
     Algebraic x_diff = x1 - a, y_diff = y1 - b;
     // the coefficients are:
     Algebraic A = 1,
-              B = -2*c,
+              B = Rational(-2*c),
```

Comment:
To write `Rational` to fix CGAL code is not really a problem, but it worries me for user code.
I have put an [issue ](https://github.com/boostorg/multiprecision/issues/541) in the boost multiprecision project on github.

Reply:
https://github.com/boostorg/multiprecision/issues/541 is another reason to require C++17 for CGAL-5.6.

Reply:
Actually, that is the fix https://github.com/boostorg/multiprecision/pull/543 that will require C++17, in Boost 1.82.

Reply:
It is one thing to require C++17, but a completely different thing  to require Boost 1.82 as minimal version for CGAL. 
The fix is not some code like a traits class or a free function that could be packaged separately to fix older versions of Boost. 
So maybe the idea to just `typedef  boost::mp::cpp_int BigInt`   is not viable and we still need  a wrapper class around it.   Which is a pity as we would not profit from expression templates.